pinot-common/src/main/java/org/apache/pinot/common/function/scalar/JsonFunctions.java

```diff
@@ -66,7 +66,7 @@ private JsonFunctions() {
   /**
    * Convert Map to Json String
    */
-  @ScalarFunction
+  @ScalarFunction(nullableParameters = true)
   public static String toJsonMapStr(Map map)
```

Comment:
Let's annotate the nullable parameter as `Nullable`, same for other annotated functions